MANIFEST.in

```diff
@@ -1,4 +1,3 @@
 include README.rst AUTHORS.rst CONTRIBUTING.md LICENSE
 include lightkurve/data/*
-include lightkurve/prf/*
-include lightkurve/tests/data/*
```

Comment:
@johnnyzhang295 Can you try rebasing against master?  It looks like this PR has been merged in a way that would revert some recent commits we'd want to keep!